src/main/java/ua/softserve/rv_028/issuecitymonitor/entity/User.java

```diff
@@ -44,8 +44,7 @@
 	private String password;
 
 	@NotEmpty
-	@NaturalId
-	@Column(name = "email", unique = true)
+	@Column(name = "email")
```

Comment:
Email field must be unique to avoid the same emails in the db, also this field a NaturalId for our app

Reply:
If this column will be have unique key then I can't use method save() in DAO, and it add me new problems so I think better if our project will not have any limitation in DB but have limitation in entity.